x-pack/plugins/cases/public/components/user_actions/use_user_actions_last_page.test.tsx

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { renderHook } from '@testing-library/react-hooks';
+
+import { useLastPageUserActions } from './use_user_actions_last_page';
+import type { UserActivityParams } from '../user_actions_activity_bar/types';
+import { useFindCaseUserActions } from '../../containers/use_find_case_user_actions';
+import { defaultUseFindCaseUserActions } from '../case_view/mocks';
+import { basicCase } from '../../containers/mock';
+
+const userActivityQueryParams: UserActivityParams = {
+  type: 'all',
+  sortOrder: 'asc',
+  page: 1,
+  perPage: 10,
+};
+
+jest.mock('../../containers/use_find_case_user_actions');
+jest.mock('../../common/lib/kibana');
+
+const useFindCaseUserActionsMock = useFindCaseUserActions as jest.Mock;
+
+describe('useLastPageUserActions', () => {
```

Comment:
Can you add a test where you check that `lastPageUserActions` is an empty array when `useFindCaseUserActionsMock` returns `undefined` for data?